packages/json-prop/src/main.ts

```diff
@@ -1,10 +1,8 @@
 import * as core from '@actions/core';
-import { path } from 'ramda';
-
 import * as io from '@eventespresso-actions/io';
-import { toPath } from '@eventespresso-actions/utils';
-
 import { getInput } from './utils';
+import { path } from 'ramda';
+import { toPath } from '@eventespresso-actions/utils';
```

Comment:
This is nice I think I also need to apply this sorting to my code to easy track.

Reply:
only had to do that because eslint was freaking out about the order... 🔠 is nice though